universum/configuration_support.py

```diff
@@ -128,7 +129,7 @@ def __add__(self, other):
         """
         return Variations(list.__add__(list(self), other))
 
-    def __mul__(self, other):
+    def __mul__(self, other: Union[int, List]) -> 'Variations':
```

Comment:
Why Union[int, List], not Union[int, 'Variations']?

Reply:
The doc string has to include `int` as valid type for `other`

Reply:
I used List to align with __add__ parameters
'Variations' can always be used instead of List

Reply:
Changed to 'Variations' now

Reply:
I don't see the change.

Reply:
Please double-check whether you're reviewing the latest revision

Reply:
You are right. I didn't see the last version.